src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -118,11 +118,6 @@ public CacheHandle lookup(Spawn spawn, SpawnExecutionContext context)
       // the remote caching is disabled and the only configured cache is remote.
       return SpawnCache.NO_RESULT_NO_STORE;
     }
-    boolean checkCache = options.remoteAcceptCached;
-
-    if (checkCache) {
```

Comment:
How come you removed this?

Reply:
hmm I see that you moved the progress status down to the other block, but will the logic between here and there cause a delay?

Reply:
It passes the tests. I don't see a strong reason to keep it separate. I might be wrong. @ishikhman do you have more context on it?

Reply:
I don't see a bit difference to be honest. Should be fine both ways.